### PR TITLE
added mrot configuration

### PR DESCRIPTION
**Pull request by jayeshh123**
@rajan-mis this PR is related to MROT configuration. Request you to review it.

This PR contains MROT and Logical subnet related file and changes.

### CODE REVIEW

**Comment by rajan-mis** [comment]
@jayeshh123  General comment. 

1. Please update the description 
2. commit is not signed. 
3. Add the required read me for overall work flow and how user can consume this. 
4. Print the register output for CMD execution then it becomes easy to debug.

File: roles/mrot_config/tasks/readme [hunk 1]
```diff
@@ -0,0 +1,115 @@
+What is MROT??
```
Comment (rajan-mis) [inline roles/mrot_config/tasks/readme:1]: Please rename this file to , `README.md`

Comment (jayeshh123) [reply roles/mrot_config/tasks/readme]: Resolved. Name is update to README.md

**Comment by jayeshh123** [comment]
Raising new PR from fresh branch

**Pull request closed without merging**